taqtile/src/pages/Login/Login.css

```diff
@@ -0,0 +1,40 @@
+body{
+    background-color: rgb(62, 62, 130);
+}
+.container-login{
+    background-color: rgb(23, 109, 109);
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    padding: 80px;
+    border-radius: 15px;
+}
+
+.label-login{
+    padding: 10px;
+    font-size: 20px;
+}
+
+.input-login{
+    padding: 10px;
+    border: none;
+    outline: none;
```

Comment:
Input vem com valor de outline pré definido no seu estilo  principal?

Reply:
Não na principal, mas no arquivo css de login

Reply:
Não é isso, vc tirou a estilização do outline, mas o outline não está presente no estilo principal, ele é aplicado no seletor `focus-visible`.

É importante a gente aplicar as estilizações no seletor corretor para evitar resultados indesejados.

![image](https://github.com/user-attachments/assets/3d8c8502-10cf-46d7-88c6-762f28c06a77)


```suggestion
    .input-login:focus-visible{
    outline: none;
}
```

Reply:
Entendi, corrigi, obrigada.